src/couch_replicator_tasks.erl

```diff
@@ -0,0 +1,52 @@
+-module(couch_replicator_tasks).
+
+-export([all/0]).
+
+-include_lib("couch/include/couch_db.hrl").
+-include("couch_replicator.hrl").
+-include("couch_replicator_api_wrap.hrl").
+
+
+all() ->
```

Comment:
Moving this to the scheduler module makes sense to me. We can move it into its own module later if it grows too large.